ClientApp/src/App.tsx

```diff
@@ -1,12 +1,29 @@
 import React, { FunctionComponent } from 'react';
 import { Route } from 'react-router';
-import { Layout } from './components/Layout';
 import { Home } from './components/Home';
-
 import './custom.scss';
 
+import { Layout } from 'antd';
+import { Typography } from 'antd';
+
+const { Header, Content, Footer, Sider } = Layout;
+const { Title } = Typography;
+
 export const App: FunctionComponent = () => (
   <Layout>
-    <Route exact path='/' component={Home} />
+    <Header className="header">
+      <Title style={{color:"white"}}>Hotdish</Title>
```

Comment:
Refrain from using inline styles, since they can get out of hand quickly. Take this out to the `scss` file